multisig_wallet/multisig_wallet.py

```diff
@@ -325,6 +325,10 @@ def getTransactionsExecuted(self, _transactionId: int) -> bool:
     def checkIfWalletOwner(self, _walletOwner: Address) -> bool:
         return _walletOwner in self._wallet_owners
 
+    @external(readonly=True)
+    def getTotalNumberOfWalletOwner(self) -> int:
```

Comment:
It seems the function name is too verbose. How about using the following name that is similar to `getConfirmationCount`.
```suggestion
    def getWalletOwnerCount(self) -> int:
```